lib/sidekiq/worker.rb

```diff
@@ -218,7 +218,7 @@ def perform_inline(*args)
         msg = Sidekiq.load_json(Sidekiq.dump_json(item))
 
         # prepare the job instance
-        klass = msg["class"].constantize
```

Comment:
isn't this using the Rails `String#constantize` method?

Reply:
Yep, `perform_inline` is a relatively new addition to the API. Looks like few are using it so far.